singleheader/CMakeLists.txt

```diff
@@ -67,32 +97,34 @@ else(MSVC)
       DEPENDS ${SINGLEHEADER_FILES}
     )
   endif()
-
   #
   # "make amalgamate" to generate the header files directly and update the original source
   #
   add_custom_target(amalgamate DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/simdjson.cpp ${CMAKE_CURRENT_SOURCE_DIR}/simdjson.h ${CMAKE_CURRENT_SOURCE_DIR}/amalgamate_demo.cpp ${CMAKE_CURRENT_SOURCE_DIR}/README.md)
-  
+  add_dependencies(simdjson amalgamate)
```

Comment:
Would it be more explicit for simdjson to depend on the aggregated simdjson.h directly? That would trigger an amalgamation anyway, due to the output thing, wouldn't it?